bcda/auth/client/oktaclient.go

```diff
@@ -184,7 +247,9 @@ func addClientToPolicy(clientID string, requestID uuid.UUID) error {
 
 	// not calling logRequest() because this is a step of AddClientApplication
 
-	resp, err := client().Do(req)
+	var client = &http.Client{Timeout: time.Second * 10}
+	resp, err := client.Do(req)
```

Comment:
let's use this func i made just to reuse code and do `client().Do(req)`
https://github.com/CMSgov/bcda-app/blob/7ee08580546daba2c233a0c95048764dc2ec2cf4/bcda/auth/client/oktaclient.go#L285-L287

Reply:
as well as everywhere else we can

Reply:
Great!